bc_obps/compliance/service/compliance_obligation_service.py

```diff
@@ -71,6 +72,9 @@ def create_compliance_obligation(
             penalty_status=ComplianceObligation.PenaltyStatus.NONE,
         )
 
+        # Create initial empty penalties for this obligation
+        PenaltyCalculationService.create_initial_penalties_for_obligation(obligation)
+
```

Comment:
We don't want our service to be creating penalties here, this should only be done by our automated processes